### PR TITLE
Service representation GA: docs changes

### PR DESCRIPTION
**Pull request by dussault-antoine**
### What does this PR do? What is the motivation?

- Tracked in this [DOCS ticket](https://datadoghq.atlassian.net/browse/DOCS-8963) and this [APMST ticket](https://datadoghq.atlassian.net/browse/APMST-1270).
- The status of all docs changes required for service representation GA is tracked [here](https://docs.google.com/document/d/1gcjH9mfEcDuZmG34yJ78JAwhjJnBXUbEuKWIktp055g/edit?tab=t.0)

### Merge instructions

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: content/en/tracing/services/inferred_services.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+{{< code-block lang="yaml" filename="datadog.yaml" collapsible="true" >}}
+
+apm_config:
+  compute_stats_by_span_kind: true
+  peer_tags_aggregation: true
+
+{{< /code-block >}}
+
+Alternatively, configure this by setting the following environment variables in your Datadog Agent launch configuration:
```
Comment (brett0000FF) [inline content/en/tracing/services/inferred_services.md:34]:
```suggestion
Alternatively, set these environment variables in your Datadog Agent configuration:
```

File: content/en/tracing/services/inferred_services.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+{{< code-block collapsible="true" lang="yaml" >}}
+
+DD_APM_COMPUTE_STATS_BY_SPAN_KIND=true 
+DD_APM_PEER_TAGS_AGGREGATION=true
+
+{{< /code-block >}}
+
+If you are using Helm, nclude the same set of environment variables in your `values.yaml` [file][2].
```
Comment (brett0000FF) [inline content/en/tracing/services/inferred_services.md:43]:
```suggestion
If you are using Helm, include these environment variables in your `values.yaml` [file][2].
```

File: content/en/tracing/services/inferred_services.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+[1]: https://github.com/DataDog/datadog-agent/releases/tag/7.55.1
+[2]: https://github.com/DataDog/helm-charts/blob/main/charts/datadog/values.yaml
+{{% /tab %}}
+{{% tab "Agent version between 7.50.3 and 7.54.1" %}}
```
Comment (brett0000FF) [inline content/en/tracing/services/inferred_services.md:48]:
```suggestion
{{% tab "Agent v7.50.3 - v7.54.1" %}}
```

File: content/en/tracing/services/inferred_services.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+
+If you use a Datadog Agent version >= [7.50.3][1] and <= [7.54.1][2], update your `datadog.yaml` configuration file with the following:
```
Comment (brett0000FF) [inline content/en/tracing/services/inferred_services.md:50]:
```suggestion
For Datadog Agent versions [7.50.3][1] through [7.54.1][2], add the following to your `datadog.yaml` configuration file:
```

File: content/en/tracing/services/inferred_services.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+{{< code-block lang="yaml" filename="datadog.yaml" collapsible="true" >}}
+
+apm_config:
+  compute_stats_by_span_kind: true
+  peer_tags_aggregation: true
+  peer_tags: ["_dd.base_service","amqp.destination","amqp.exchange","amqp.queue","aws.queue.name","aws.s3.bucket","bucketname","cassandra.keyspace","db.cassandra.contact.points","db.couchbase.seed.nodes","db.hostname","db.instance","db.name","db.namespace","db.system","grpc.host","hostname","http.host","http.server_name","messaging.destination","messaging.destination.name","messaging.kafka.bootstrap.servers","messaging.rabbitmq.exchange","messaging.system","mongodb.db","msmq.queue.path","net.peer.name","network.destination.name","peer.hostname","peer.service","queuename","rpc.service","rpc.system","server.address","streamname","tablename","topicname"]
+
+{{< /code-block >}}
+
+Alternatively, configure this by setting the following environment variables in your Datadog Agent launch configuration:
```
Comment (brett0000FF) [inline content/en/tracing/services/inferred_services.md:61]:
```suggestion
Alternatively, set these environment variables in your Datadog Agent configuration:
```

File: content/en/tracing/services/inferred_services.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+
+{{< code-block collapsible="true" lang="yaml" >}}
+
+DD_APM_COMPUTE_STATS_BY_SPAN_KIND=true 
+DD_APM_PEER_TAGS_AGGREGATION=true
+DD_APM_PEER_TAGS='["_dd.base_service","amqp.destination","amqp.exchange","amqp.queue","aws.queue.name","aws.s3.bucket","bucketname","cassandra.keyspace","db.cassandra.contact.points","db.couchbase.seed.nodes","db.hostname","db.instance","db.name","db.namespace","db.system","grpc.host","hostname","http.host","http.server_name","messaging.destination","messaging.destination.name","messaging.kafka.bootstrap.servers","messaging.rabbitmq.exchange","messaging.system","mongodb.db","msmq.queue.path","net.peer.name","network.destination.name","peer.hostname","peer.service","queuename","rpc.service","rpc.system","server.address","streamname","tablename","topicname"]'
+
+{{< /code-block >}}
+
+If you are using Helm, nclude the same set of environment variables in your `values.yaml` [file][3].
```
Comment (brett0000FF) [inline content/en/tracing/services/inferred_services.md:71]:
```suggestion
If you are using Helm, include these environment variables in your `values.yaml` [file][3].
```

File: content/en/tracing/services/inferred_services.md [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+
+[1]: https://github.com/DataDog/datadog-agent/releases/tag/7.50.3
+[2]: https://github.com/DataDog/datadog-agent/releases/tag/7.54.1
+[3]: https://github.com/DataDog/helm-charts/blob/main/charts/datadog/values.yaml
+{{% /tab %}}
+{{% tab "OpenTelemetry collector" %}}
```
Comment (brett0000FF) [inline content/en/tracing/services/inferred_services.md:77]:
```suggestion
{{% tab "OpenTelemetry Collector" %}}
```

File: content/en/tracing/services/inferred_services.md [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+
+If you are using the OpenTelemetry collector, the minimum version recommended is the opentelemetry-collector-contrib >= [v0.95.0][1]. Update the collector configuration with:
```
Comment (brett0000FF) [inline content/en/tracing/services/inferred_services.md:79]:
```suggestion
For the OpenTelemetry Collector, the minimum recommended version is `opentelemetry-collector-contrib` [v0.95.0][1] or later. In that case, update this configuration:
```

File: content/en/tracing/services/inferred_services.md [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+connectors:
+  datadog/connector:
+    traces:
+      compute_stats_by_span_kind: true
+      peer_tags_aggregation: true
+      peer_tags: ["_dd.base_service","amqp.destination","amqp.exchange","amqp.queue","aws.queue.name","aws.s3.bucket","bucketname","db.cassandra.contact.points","db.couchbase.seed.nodes","db.hostname","db.instance","db.name","db.namespace","db.system","grpc.host","hostname","http.host","http.server_name","messaging.destination","messaging.destination.name","messaging.kafka.bootstrap.servers","messaging.rabbitmq.exchange","messaging.system","mongodb.db","msmq.queue.path","net.peer.name","network.destination.name","peer.hostname","peer.service","queuename","rpc.service","rpc.system","server.address","streamname","tablename","topicname"]
+
+{{< /code-block >}}
+
+If your collector version is below v0.95.0, update the collector configuration with :
```
Comment (brett0000FF) [inline content/en/tracing/services/inferred_services.md:92]:
```suggestion
If your Collector version is below v0.95.0, update the following Collector configuration:
```

File: content/en/tracing/services/inferred_services.md [hunk 10]
```diff
@@ -0,0 +1,153 @@
...
+**Example**: [collector.yaml][2].
+
+[1]: https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.95.0
+[2]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/datadogexporter/examples/collector.yaml#L335-L357
+{{% /tab %}}
+{{< /tabs >}}
+
+## Naming inferred entities
+
+To determine the names and types of the inferred service dependencies, Datadog uses standard span attributes and maps them to `peer.*` attributes. For example, inferred external APIs use the default naming scheme `net.peer.name`, e.g. `api.stripe.com`, `api.twilio.com`, `us6.api.mailchimp.com`. Inferred databases use the default naming scheme `db.instance`.
```
Comment (brett0000FF) [inline content/en/tracing/services/inferred_services.md:114]:
```suggestion
To determine the names and types of the inferred service dependencies, Datadog uses standard span attributes and maps them to `peer.*` attributes. For example, inferred external APIs use the default naming scheme `net.peer.name` like `api.stripe.com`, `api.twilio.com`, and `us6.api.mailchimp.com`. Inferred databases use the default naming scheme `db.instance`.
```

File: content/en/tracing/services/inferred_services.md [hunk 11]
```diff
@@ -0,0 +1,153 @@
...
+
+### List of newly introduced peer.* tags
```
Comment (brett0000FF) [inline content/en/tracing/services/inferred_services.md:116]:
```suggestion
### Peer tags
```

File: content/en/tracing/services/inferred_services.md [hunk 12]
```diff
@@ -0,0 +1,153 @@
...
+{{% /tab %}}
+{{< /tabs >}}
+
+## Naming inferred entities
+
+To determine the names and types of the inferred service dependencies, Datadog uses standard span attributes and maps them to `peer.*` attributes. For example, inferred external APIs use the default naming scheme `net.peer.name`, e.g. `api.stripe.com`, `api.twilio.com`, `us6.api.mailchimp.com`. Inferred databases use the default naming scheme `db.instance`.
+
+### List of newly introduced peer.* tags 
+
+`peer.*` dimensions | Remapped from ...
```
Comment (brett0000FF) [inline content/en/tracing/services/inferred_services.md:118]:
```suggestion
Peer Tag | Source Attributes
```

File: content/en/tracing/services/inferred_services.md [hunk 13]
```diff
@@ -0,0 +1,153 @@
...
+`peer.db.system` | `db.system`
+`peer.hostname` | `peer.hostname`, `hostname`, `net.peer.name`, `db.hostname`, `network.destination.name`, `grpc.host`, `http.host`, `server.address`, `http.server_name`
+`peer.kafka.bootstrap.servers` | `messaging.kafka.bootstrap.servers`
+`peer.messaging.destination` | `topicname`, `messaging.destination`, `messaging.destination.name`, `messaging.rabbitmq.exchange`, `amqp.destination`, `amqp.queue`, `amqp.exchange`, `msmq.queue.path`, `aws.queue.name`
+`peer.messaging.system` | `messaging.system`
+`peer.rpc.service` | `rpc.service`
+`peer.rpc.system` | `rpc.system`
+`peer.service` | `peer.service`
+
+## Global default service naming migration
```
Comment (brett0000FF) [inline content/en/tracing/services/inferred_services.md:136]:
```suggestion
## Migrate to global default service naming
```

File: content/en/tracing/services/inferred_services.md [hunk 14]
```diff
@@ -0,0 +1,153 @@
...
+
+With inferred services, service dependencies are automatically detected from existing span attributes. As a result, changing service names (via the `service` tag) is not required anymore to identify these dependencies.
```
Comment (brett0000FF) [inline content/en/tracing/services/inferred_services.md:138]:
```suggestion
With inferred services, service dependencies are automatically detected from existing span attributes. As a result, changing service names (using the `service` tag) is not required to identify these dependencies. 
```

File: content/en/tracing/services/inferred_services.md [hunk 15]
```diff
@@ -0,0 +1,153 @@
...
+`peer.messaging.system` | `messaging.system`
+`peer.rpc.service` | `rpc.service`
+`peer.rpc.system` | `rpc.system`
+`peer.service` | `peer.service`
+
+## Global default service naming migration
+
+With inferred services, service dependencies are automatically detected from existing span attributes. As a result, changing service names (via the `service` tag) is not required anymore to identify these dependencies. 
+
+Enabling `DD_TRACE_REMOVE_INTEGRATION_SERVICE_NAMES_ENABLED` environment variable ensures that no Datadog integration sets service names that are different from the default global service name. It improves how service-to-service connections and inferred services are represented in Datadog visualizations, across all supported tracing library languages and integrations.
```
Comment (brett0000FF) [inline content/en/tracing/services/inferred_services.md:140]:
```suggestion
Enable `DD_TRACE_REMOVE_INTEGRATION_SERVICE_NAMES_ENABLED` to ensures no Datadog integration sets service names that are different from the default global service name. This also improves how service-to-service connections and inferred services are represented in Datadog visualizations, across all supported tracing library languages and integrations.
```

File: content/en/tracing/services/resource_page.md [hunk 16]
```diff
@@ -50,27 +50,19 @@ Use the top right percentile selectors to zoom into a given percentile, or hover
...
 
-You can also view a map of all of a resource's upstream and downstream service dependencies. With the Dependency Map Navigator, you can see the flow of services, with spans that go through a specific resource (endpoint, database query, etc.) end-to-end, along with their request counts.
+Use the Dependency Map to view a flow graph of all of a resource's upstream and downstream service dependencies.  The map is scoped to the requests flowing through the selected service and resource  (endpoint, database query, etc.) you're focused on. [Inferred service dependencies][10] like databases, queues or third-party services are represented with a purple background node.
 
-This map is based on a sample of ingested spans; the sample is drawn by a fixed sampling algorithm that considers the structure of traces. The sampling algorithm is not configurable and is not impacted by ingestion control.
+Click on a downstream or upstream service node to see which resources are invoked in the request flow. To focus on a particular request path, select a node an click `set as start/end`. The map will be filtered to focus on the requests that also flow through this upstream or downstream dependency.
```
Comment (brett0000FF) [inline content/en/tracing/services/resource_page.md:57]:
```suggestion
Click on a downstream or upstream service node to see which resources are invoked in the request flow. To focus on a particular request path, select a node an click `set as start/end`. This filters the map to focus on the requests that also flow through this upstream or downstream dependency.
```

File: content/en/tracing/services/service_page.md [hunk 17]
```diff
@@ -137,6 +137,30 @@ Choose what to display in your resources list:
 
 {{< img src="tracing/visualization/service/resource_columns.png" alt="Resource columns" style="width:40%;">}}
 
+{{< site-region region="ap1" >}}
+## Dependencies
+
+Visualize upstream and downstream dependencies that the service interacts with from the dependency map. The map is powered by [APM metrics][1], hence surfaces accurate request counts, error rates and latency numbers. The map automatically groups dependencies together by operation name. For instance, if a service calls two downstream services via gRPC, these will be displayed grouped together. The table on the left-hand side of the map shows requests and error rates over time, useful to identify failing dependencies.
```
Comment (brett0000FF) [inline content/en/tracing/services/service_page.md:143]:
```suggestion
Visualize upstream and downstream dependencies that the service interacts with from the dependency map. The map is powered by [APM metrics][1] to surface accurate request counts, error rates, and latency numbers. The map automatically groups dependencies by operation name. For instance, if a service calls two downstream services using gRPC, these services are grouped together. The table on the left-hand side of the map shows requests and error rates over time, useful to identify failing dependencies.
```

File: content/en/tracing/services/service_page.md [hunk 18]
```diff
@@ -137,6 +137,30 @@ Choose what to display in your resources list:
...
+
+[Inferred service dependencies][2] like databases, queues or third-party services are represented with a purple background node.
```
Comment (brett0000FF) [inline content/en/tracing/services/service_page.md:145]:
```suggestion
[Inferred service dependencies][2] like databases, queues, or third-party services are represented with a purple background node.
```

File: content/en/tracing/services/service_page.md [hunk 19]
```diff
@@ -137,6 +137,30 @@ Choose what to display in your resources list:
...
+
+{{< img src="tracing/visualization/service/dependencies_section.png" alt="Service page dependency map" style="width:100%;">}}
+
+**Note**: [service overrides][3] are represented as part of the edge of the dependency map to keep visibility over the actual remote service, database or queue the service is interacting with.
```
Comment (brett0000FF) [inline content/en/tracing/services/service_page.md:149]:
```suggestion
**Note**: [Service overrides][3] display along the edges (connecting lines) between nodes in the dependency map to keep visibility over the actual remote service, database, or queue the service is interacting with.
```

File: content/en/tracing/services/service_page.md [hunk 20]
```diff
@@ -137,6 +137,30 @@ Choose what to display in your resources list:
...
+
+### Investigating a specific service dependency
+
+Click on a dependency service node to investigate one of the services' dependencies. In the side panel, request count, errors and latency metrics will be scoped to requests going from the service to the downstream dependency.
```
Comment (brett0000FF) [inline content/en/tracing/services/service_page.md:153]:
```suggestion
Click on a dependency service node to investigate one of the service's dependencies. In the side panel, request count, errors, and latency metrics are scoped to requests going from the service to the downstream dependency.
```

File: content/en/tracing/services/service_page.md [hunk 21]
```diff
@@ -137,6 +137,30 @@ Choose what to display in your resources list:
...
+
+Use the table below to see which resources are invoked as part of the client call going from the service to the downstream dependency. For instance, if the service is calling a database, see the breakdown of queries that are made to the database, and use the table to identify slow or failing queries.
```
Comment (brett0000FF) [inline content/en/tracing/services/service_page.md:155]:
```suggestion
Use the following table to see which resources are invoked in client calls from the service to the downstream dependency. For instance, if the service is calling a database, see the breakdown of queries that are made to the database, and use the table to identify slow or failing queries.
```

File: content/en/tracing/services/service_page.md [hunk 22]
```diff
@@ -137,6 +137,30 @@ Choose what to display in your resources list:
 
 {{< img src="tracing/visualization/service/resource_columns.png" alt="Resource columns" style="width:40%;">}}
 
+{{< site-region region="ap1" >}}
+## Dependencies
```
Comment (brett0000FF) [inline content/en/tracing/services/service_page.md:141]: Note: We have a link to this section from **Overview**. We should wrap that in the site-region shortcode too so users outside of AP1 don't get confused:

**Review by brett0000FF — COMMENTED** [review]
Looking good so far! Let me know if you have any questions about my edits or comments.

File: content/en/tracing/trace_explorer/span_tags_attributes.md [hunk 23]
```diff
@@ -9,25 +11,35 @@ further_reading:
...
-- Perform trace analytics
-- Start troubleshooting once your spans are ingested
+* **[Span tags](#span-tags)** are enrichments of context related to the span, for instance host or container tags describing the infrastructure the service is running on. 
 
-{{< img src="tracing/trace_explorer/facets/facet_panel.png" alt="The Facets panel in the Trace Explorer" style="width:90%;">}}
+You can query spans by any span tag and attribute from the [trace explorer][6].
+
+### Reserved attributes
+
+Reserved attributes are a subset of span attributes that are present on every span. These attributes are queriable without prepending the `@` character. The full list of reserved attributes is: `env`, `service`, `operation_name`, `resource_name`, `status`, `ingestion_reason`, `trace_id`. Refer to the [APM terms and concepts][10] for a full definition of these terms.
```
Comment (brett0000FF) [inline content/en/tracing/trace_explorer/span_tags_attributes.md:24]:
```suggestion
Reserved attributes are a subset of span attributes that are present on every span. These attributes are queryable without prepending the `@` character. The full list of reserved attributes is: `env`, `service`, `operation_name`, `resource_name`, `status`, `ingestion_reason`, `trace_id`. Refer to the [APM terms and concepts][10] for a full definition of these terms.
```

File: content/en/tracing/trace_explorer/span_tags_attributes.md [hunk 24]
```diff
@@ -9,25 +11,35 @@ further_reading:
...
+
+### Span attributes
+
+Span attributes are the content of your span. These are collected out-of-the-box in tracing libraries via auto-instrumentation, manually via custom instrumentation, or remapped in Datadog backend based on source attributes (c.f. [peer attributes][11], remapped from some source attributes). To search on a specific span attribute you must prepend an `@` character at the beginning of the attribute key.
```
Comment (brett0000FF) [inline content/en/tracing/trace_explorer/span_tags_attributes.md:28]:
```suggestion
Span attributes are the content of your span. These are collected out-of-the-box in tracing libraries using automatic instrumentation, manually using custom instrumentation, or remapped in the Datadog backend based on source attributes (see [peer attributes][11], remapped from some source attributes). To search on a specific span attribute, you must prepend an `@` character at the beginning of the attribute key.
```

File: content/en/tracing/trace_explorer/span_tags_attributes.md [hunk 25]
```diff
@@ -9,25 +11,35 @@ further_reading:
...
 
-[1]: /tracing/trace_explorer/search
-[2]: /tracing/trace_pipeline/generate_metrics
-[3]: /tracing/trace_pipeline/trace_retention/#retention-filters
+Span tags are context around your span, enriched based on the host or the container the service the span is emitted from is running on. You don't need to prepend an `@` character to query for span tags.
 
-{{< /site-region >}}
+## Facets
+
+You can create **facets** on top of span tags and attributes to map an attribute to the right type (e.g. string or int) and for these attributes to show up in the facet list.
```
Comment (brett0000FF) [inline content/en/tracing/trace_explorer/span_tags_attributes.md:38]:
```suggestion
You can create **facets** on top of span tags and attributes to map an attribute to the right type (for example, string or int) and for these attributes to show up in the facet list.
```

File: content/en/tracing/trace_explorer/trace_view.md [hunk 26]
```diff
@@ -137,19 +146,40 @@ The span header contains service, operation, and resource names of the selected
...
+- Spark Metrics (on Spark job spans)
 - A git warning message (when git information is missing on a CI Test)
-- SQL Query markup (on a SQL query)
-- RUM Context and Metadata (on a RUM span)
-- Spark Metrics (on a Spark job span)
 
 {{< img src="tracing/trace_view/info_tab.png" alt="Span Info tab" style="width:90%;">}}
 
+{{< site-region region="ap1,us3,us5" >}}
+When the service name is an override from the base service name, the top of the info sections shows:
```
Comment (brett0000FF) [inline content/en/tracing/trace_explorer/trace_view.md:170]:
```suggestion
When the service name is an override from the base service name, the top of the info section shows the:
```

File: content/en/tracing/trace_explorer/trace_view.md [hunk 27]
```diff
@@ -137,19 +146,40 @@ The span header contains service, operation, and resource names of the selected
...
+- the **[base service][2]**: service from which the span is emitted, identified by the `@base_service` attribute.
+- the **[service override][3]**: service name, different from the base service name, set automatically in Datadog integrations or changed via the programmatic API. The service override is identified by the `service` reserved attribute.
+- the **[inferred service][4]** (_when applicable_): name of the inferred entity being called by the base service, identified by one of the [peer attributes][5]
```
Comment (brett0000FF) [inline content/en/tracing/trace_explorer/trace_view.md:173]:
```suggestion
- **[Base service][2]**: service from which the span is emitted, identified by the `@base_service` attribute.
- **[Service override][3]**: service name, different from the base service name, set automatically in Datadog integrations or changed via the programmatic API. The service override is identified by the `service` reserved attribute.
- **[Inferred service][4]** (_when applicable_): name of the inferred entity being called by the base service, identified by one of the [peer attributes][5].
```

**Review by brett0000FF — COMMENTED** [review]
Just a few comments on the latest commits!

File: content/en/tracing/services/service_page.md (hunk 22 above)
Comment (dussault-antoine) [reply content/en/tracing/services/service_page.md:143]: Done !

**Review by brett0000FF — APPROVED** [review]
🚀

**Pull request merged**